MapboxNavigation/Style.swift

```diff
@@ -222,87 +200,161 @@ public class Style: NSObject {
+        if let font = distanceLabelFont {
+            DistanceLabel.appearance(for: traitCollection).font = font.adjustedFont.with(fontFamily: fontFamily)
+        }
+        
+        if let color = distanceLabelTextColor {
+            DistanceLabel.appearance(for: traitCollection).textColor = color
+        }
+        
+        if let font = destinationLabelFont {
+            DestinationLabel.appearance(for: traitCollection).font = font.adjustedFont.with(fontFamily: fontFamily)
+        }
+        
+        if let color = destinationLabelTextColor {
+            DestinationLabel.appearance(for: traitCollection).textColor = color
+        }
+        
         // Table view (drawer)
         
         if let color = headerBackgroundColor {
             RouteTableViewHeaderView.appearance(for: traitCollection).backgroundColor = color
         }
         
+        if let color = timeRemainingLabelTextColor {
+            TimeRemainingLabel.appearance(for: traitCollection).textColor = color
+        }
+        
+        if let font = timeRemainingLabelFont {
+            TimeRemainingLabel.appearance(for: traitCollection).font = font.adjustedFont.with(fontFamily: fontFamily)
+        }
+        
+        if let color = distanceRemainingLabelTextColor {
+            DistanceRemainingLabel.appearance(for: traitCollection).textColor = color
+        }
+        
+        if let font = distanceRemainingLabelFont {
+            DistanceRemainingLabel.appearance(for: traitCollection).font = font.adjustedFont.with(fontFamily: fontFamily)
+        }
+        
+        if let color = etaLabelTextColor {
+            ETALabel.appearance(for: traitCollection).textColor = color
+        }
+        
+        if let font = etaLabelFont {
+            ETALabel.appearance(for: traitCollection).font = font.adjustedFont.with(fontFamily: fontFamily)
+        }
+        
         if let font = cellTitleLabelFont {
-            CellTitleLabel.appearance(for: traitCollection).font = font
+            CellTitleLabel.appearance(for: traitCollection).font = font.adjustedFont.with(fontFamily: fontFamily)
         }
         
         if let color = cellTitleLabelTextColor {
             CellTitleLabel.appearance(for: traitCollection).textColor = color
         }
         
         if let font = cellSubtitleLabelFont {
-            CellSubtitleLabel.appearance(for: traitCollection).font = font
+            CellSubtitleLabel.appearance(for: traitCollection).font = font.adjustedFont.with(fontFamily: fontFamily)
         }
         
         if let color = cellSubtitleLabelTextColor {
             CellSubtitleLabel.appearance(for: traitCollection).textColor = color
         }
-        
-        if let color = wayNameTextColor {
-            WayNameLabel.appearance(for: traitCollection).textColor = color
-        }
-        
-        if let font = wayNameLabelFont {
-            WayNameLabel.appearance(for: traitCollection).font = font
-        }
     }
 }
 
 /**
+ :nodoc:
```

Comment:
I hesitated a while regarding this. It's just `UILabel` with no additional functionality but it might be useful if you want to do some custom appearance styling to at least expose the class?